src/tree_store/page_store/file_lock/unix.rs

```diff
@@ -1,14 +1,33 @@
+// TODO once Rust's libc has flock implemented for WASI, this file needs to be revisited.
+// What needs to be changed is commented below.
+// See also: https://github.com/WebAssembly/wasi-filesystem/issues/2
+
+// Remove this line once wasi-libc has flock
+#![cfg_attr(feature = "wasi", allow(unused_imports))]
```

Comment:
This silences some unused import warnings that will go away once wasi-libc has flock